llms/mlx_lm/models/qwen2_moe.py

```diff
@@ -0,0 +1,258 @@
+from dataclasses import dataclass
+from typing import Dict, Optional, Tuple, Union
+
+import mlx.core as mx
+import mlx.nn as nn
+import numpy as np
+
+from .base import BaseModelArgs
+
+
+@dataclass
+class ModelArgs(BaseModelArgs):
+    model_type: str
+    hidden_size: int
+    num_hidden_layers: int
+    intermediate_size: int
+    num_attention_heads: int
+    num_experts_per_tok: int
+    num_experts: int
+    decoder_sparse_step: int
```

Comment:
This argument is not used anywhere as far as I can tell. Can you remove?

Reply:
Removed it ✅ 

It was being used in my version but now it's not.